docker-compose.yml

```diff
@@ -16,16 +16,16 @@ networks:
 services:
   sia:
     build:
-      context: ./docker/sia
-      dockerfile: Dockerfile
+      context: https://github.com/SkynetLabs/docker-skyd.git#main
+      dockerfile: scratch/Dockerfile
       args:
         branch: portal-latest
+    command: --disable-api-security --api-addr :9980 --modules gctwra
```

Comment:
What is `a`? It's not documented [here](https://github.com/NebulousLabs/Sia/blob/master/cmd/siad/main.go#L73-L131)

Reply:
It's accounting. Documented here: https://gitlab.com/SkynetLabs/skyd/-/blob/master/cmd/skyd/main.go#L148

The repo on GitHub is out of date, the main repo is on GitLab. We need to setup auto-sync.